Dockerfile.dapper

```diff
@@ -15,7 +15,7 @@ ENV HELM_VERSION v3.3.1
 ENV KUSTOMIZE_VERSION v3.5.4
```

Comment:
RKE is missing: https://github.com/rancher/rancher/blob/release/v2.5/go.mod#L102
Make sure whole release checklist step 4 is done.

Reply:
Looks like KDM stuff missing too: https://github.com/rancher/rancher/blob/release/v2.5/pkg/settings/setting.go#L225